index.php

```diff
@@ -1,3 +1,13 @@
+<?php
+
+require_once 'php/dbConnection.php';
+require_once 'php/viewMeFunctions.php';
+$db = getDbConn();
+$viewAboutMe = viewAboutMe($db);
+$paragraph = getParagraph($viewAboutMe);
```

Comment:
Could potentially give this a slightly more descriptive name 'aboutMeParagraph'?